### ISSUE TITLE
AttributeError: module 'jsonref' has no attribute 'JsonLoader'

### ISSUE THREAD

**Issue by duncandewhurst** [OpenDataServices/flatten-tool#409]
To reproduce, follow the [installation instructions](https://flatten-tool.readthedocs.io/en/latest/getting-started/#installation) then run `flatten-tool`.

Traceback:

```
  File "/home/ddewhurst/.pyenv/versions/flatten-tool/bin/flatten-tool", line 7, in <module>
    exec(compile(f.read(), __file__, 'exec'))
  File "/home/ddewhurst/open_data_services/repositories/co-op/flatten-tool/flatten-tool", line 2, in <module>
    import flattentool.cli
  File "/home/ddewhurst/open_data_services/repositories/co-op/flatten-tool/flattentool/__init__.py", line 8, in <module>
    from flattentool.json_input import JSONParser
  File "/home/ddewhurst/open_data_services/repositories/co-op/flatten-tool/flattentool/json_input.py", line 26, in <module>
    from flattentool.schema import make_sub_sheet_name
  File "/home/ddewhurst/open_data_services/repositories/co-op/flatten-tool/flattentool/schema.py", line 100, in <module>
    class JsonLoaderLocalRefsDisabled(jsonref.JsonLoader):
AttributeError: module 'jsonref' has no attribute 'JsonLoader'
```

I think it's due to this issue: https://github.com/gazpachoking/jsonref/issues/45#issuecomment-1295312114

**Comment by duncandewhurst** [comment]
A quick fix would be to require jsonref<1, however that is incompatible with [JSCC](https://jscc.readthedocs.io/en/latest/index.html), which requires jsonref>=1.

**Comment by duncandewhurst** [comment]
Ignore me, looks like the quick fix was done in https://github.com/OpenDataServices/flatten-tool/releases/tag/v0.18.1 and I somehow didn't have the latest version.

**Comment by jpmckinney** [comment]
And there's a better fix in 0.19 that allows any version of jsonref ;)
